modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteClusterSnapshotSelfTest.java

```diff
@@ -330,6 +330,10 @@ public void testClusterSnapshotConsistencyUnderLoad() throws Exception {
 
         startGridsWithCache(grids, clientsCnt, key -> new Account(key, balance), eastCcfg, westCcfg);
 
+        // To prevent encrypted cache creation from client, we create it on the server side.
```

Comment:
Don't found such restriction in docs. What is a reason?

Reply:
Because: `IgniteSpiException: Joining node has encrypted caches which are not presented on the cluster, encrypted caches configured on client node cannot be started when such node joins the cluster, these caches can be started manually (dynamically) after node joined`